Tools/clinic/clinic.py

```diff
@@ -28,11 +28,12 @@
 import textwrap
 import traceback
 
-from collections.abc import Callable
+from collections.abc import Callable, Iterable, Reversible
 from types import FunctionType, NoneType
 from typing import (
     Any,
     Final,
+    Generator,
```

Comment:
Nit: `Generator` can also be imported from `collections.abc` rather than `typing` :)

Reply:
Yes! _That_ I actually knew 😄 Thanks!